### ISSUE TITLE
`inspect.formatargspec` is deprecated since Python 3.5

### ISSUE THREAD

**Issue by bgrant** [newrelic/newrelic-python-agent#476]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

Running with warnings turned on (`-Wa`) shows that a deprecated function is being used: 

```
newrelic/console.py: DeprecationWarning: `formatargspec` is deprecated since Python 3.5. Use `signature` and the    `Signature` object directly
```
(edited)

**Description**
[NOTE]: # ( Describe the problem you're encountering. )
`console.py` uses the deprecated `inspect.formatargspec` function.

**Expected Behavior**
[NOTE]: # ( Tell us what you expected to happen. )

I would like to be able to run with warnings turned on and not see any.

**[Troubleshooting](https://discuss.newrelic.com/t/troubleshooting-frameworks/108787) or [NR Diag](https://docs.newrelic.com/docs/using-new-relic/cross-product-functions/troubleshooting/new-relic-diagnostics) results**
[NOTE]: # ( Provide any other relevant log data. )
[TIP]:  # ( Scrub logs and diagnostic information for sensitive information )

**Steps to Reproduce**
[NOTE]: # ( Please be as specific as possible. )
[TIP]:  # ( Link a sample application that demonstrates the issue. )

**Your Environment**
[TIP]:  # ( Include as many relevant details about your environment as possible including the running version of New Relic software and any relevant configurations. )

**Additional context**
[TIP]:  # ( Add any other context about the problem here. For example, relevant community posts or support tickets. )

**Comment by umaannamalai** [comment]
Hi @bgrant! What agent version are you using? This warning should have been addressed in [v6.4.2.159](https://docs.newrelic.com/docs/release-notes/agent-release-notes/python-release-notes/python-agent-60402159#bug-fixes).

**Comment by bgrant** [comment]
You're right- this seems to have gone away after upgrading to 7.4.  Thanks!
